src/quo2/components/avatars/user_avatar/view.cljs

```diff
@@ -1,72 +1,73 @@
 (ns quo2.components.avatars.user-avatar.view
-  (:require [clojure.string :as string]
-            [quo2.components.avatars.user-avatar.style :as style]
+  (:require [quo2.components.avatars.user-avatar.style :as style]
             [quo2.components.markdown.text :as text]
             [react-native.core :as rn]
             [react-native.fast-image :as fast-image]))
 
-(defn trim-whitespace [s] (string/join " " (string/split (string/trim s) #"\s+")))
+(defn user-avatar
+  "Render user avatar with `profile-picture`
+  `profile-picture` should be one of {:uri profile-picture-uri} or {:fn profile-picture-fn}
```

Comment:
could you please give an example when uri is used and when fn

Reply:
You mean add to the doc or explain here.
There's actually no usage of `:uri` in the current codebase. 
It's I think just the component should has the ability render simple profile-picture with uri. On the usage side, since all avatar are generated by status-go so at least `port` should be provided, thus all use case in current codebase is profile-picture with :fn

Reply:
> There's actually no usage of `:uri` in the current codebase.

why is it needed then?